projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/data-job/pages/executions/model/data-job-execution.ts

```diff
@@ -38,9 +38,9 @@ export class DataJobExecutionToGridDataJobExecution {
         }
     }
 
-    static convertToDataJobExecution(
+    static convertToDataJobExecution = (
```

Comment:
Why changing it from method to class field function?
Intercepting and spying on methods in unit tests is much more easier.

If it is declared with getter and setter would be easier to spy in unit tests, but field declared like your change I think is not possible to spy on.